test/cypress/stubs/gateway-account-stubs.js

```diff
@@ -212,7 +212,19 @@ function patchAccountEmailCollectionModeSuccess (opts) {
     request: gatewayAccountFixtures.validGatewayAccountEmailCollectionModeRequest('MANDATORY')
   })
 }
+function patchAccountUpdateApplePaySuccess (gatewayAccountId, allowApplePay) {
+  const path = `/v1/api/accounts/${gatewayAccountId}`
+  return stubBuilder('PATCH', path, 200, {
+    request: gatewayAccountFixtures.validUpdateToggleApplePayRequest(allowApplePay)
+  })
+}
 
+function patchAccountUpdateGooglePayPaySuccess (gatewayAccountId, allowGooglePay) {
```

Comment:
```suggestion
function patchAccountUpdateGooglePaySuccess (gatewayAccountId, allowGooglePay) {
```

Reply:
Good spot. I've got another small PR coming so will fix this there.